lib/vdsm/tool/configurators/multipath.py

```diff
@@ -193,7 +193,7 @@
 }
 
 # Blacklist local and obsolete protocols which run on devices which are not
-# multipathable.
+# multipathable and RADOS Block Device (RBD) devices.
```

Comment:
This will be better more generic so we don't have to change it again.

   Blacklist local devices, obsolete protocols, and device nodes which
   should not be used with multipath.

Reply:
Ack